### ISSUE TITLE
Error running Forest_fire

### ISSUE THREAD

**Issue by sbacelar** [kavir1698/Agents.jl#9]
Another error:
When I do
```
forest = model_initiation(f=0.05, d=0.8, p=0.01, griddims=(20, 20), seed=2)
data = step!(dummystep, forest_step!, forest, 10, agent_properties, collect(1:10))
for i in 1:10
  visualize_2D_agent_distribution(data, forest, Symbol("pos_$i"), types=Symbol("status_$i"), savename="step_$i", cc=Dict(true=>"green", false=>"red"))
end
```
I get
```
ArgumentError: column name :pos_1 not found in the data frame

Stacktrace:
 [1] lookupname at /Users/sergiobacelar/.julia/packages/DataFrames/CZrca/src/other/index.jl:167 [inlined]
 [2] (::getfield(DataFrames, Symbol("##19#20")){DataFrames.Index})(::Symbol) at ./none:0
 [3] collect(::Base.Generator{Array{Symbol,1},getfield(DataFrames, Symbol("##19#20")){DataFrames.Index}}) at ./generator.jl:47
 [4] getindex at /Users/sergiobacelar/.julia/packages/DataFrames/CZrca/src/other/index.jl:177 [inlined]
 [5] getindex(::DataFrames.DataFrame, ::Array{Symbol,1}) at /Users/sergiobacelar/.julia/packages/DataFrames/CZrca/src/dataframe/dataframe.jl:300
 [6] #visualize_2D_agent_distribution#87(::Symbol, ::String, ::Dict{Bool,String}, ::Function, ::DataFrames.DataFrame, ::Forest{MyGrid{Int64,Array{Array{Integer,1},1}},Array{Tree,1},Float64}, ::Symbol) at /Users/sergiobacelar/.julia/packages/Agents/2wiNF/src/visualization.jl:83
 [7] (::getfield(Agents, Symbol("#kw##visualize_2D_agent_distribution")))(::NamedTuple{(:types, :savename, :cc),Tuple{Symbol,String,Dict{Bool,String}}}, ::typeof(visualize_2D_agent_distribution), ::DataFrames.DataFrame, ::Forest{MyGrid{Int64,Array{Array{Integer,1},1}},Array{Tree,1},Float64}, ::Symbol) at ./none:0
 [8] top-level scope at ./In[5]:4
```

**Comment by kavir1698** [comment]
There shouldn't be a problem after you have updated to the latest version.

**Comment by sbacelar** [comment]
I had already updated. My version of `Agents.jl` is 1.1.0 and I still need `LightGraphs` and I still get the argument error above.

**Comment by kavir1698** [comment]
Latest version is 1.1.2

**Comment by sbacelar** [comment]
Can you help me how to update `Agents`? I tried `update`. When I do `status`I get:
```
(v1.1) pkg> status "Agents"
    Status `~/.julia/environments/v1.1/Project.toml`
  [46ada45e] Agents v1.1.0
  [336ed68f] CSV v0.5.6
  [a93c6f00] DataFrames v0.18.4
  [31c24e10] Distributions v0.19.2
  [093fc24a] LightGraphs v1.2.0
  [2913bbd2] StatsBase v0.30.0
  [112f6efa] VegaLite v0.7.0
  [9a3f8284] Random
  [8dfed614] Test
```

**Comment by kavir1698** [comment]
Can you try updating from master?

`update Agents#master`

**Comment by sbacelar** [comment]
I updated from master. Now I have version 1.1.2. The problem with `LightGraphs` is solved. But the last one remains.

**Comment by sbacelar** [comment]
Problem solved with the new version of `Agents 1.1.5` and `ForestFire.jl`.

**Comment by kavir1698** [comment]
Great
